src/main/java/forestry/farming/logic/crops/CropRubber.java

```diff
@@ -83,8 +91,11 @@ public static boolean hasRubberToHarvest(IBlockState blockState) {
 	@Override
 	protected NonNullList<ItemStack> harvestBlock(World world, BlockPos pos) {
 		NonNullList<ItemStack> harvested = NonNullList.create();
-		harvested.add(PluginIC2.resin.copy());
-
+		if (PluginIC2.rubberWood != null && ItemStackUtil.equals(world.getBlockState(pos).getBlock(), PluginIC2.rubberWood)) {
```

Comment:
Store `world.getBlockState(pos).getBlock()` in a variable so it is never called twice.